parsing_scripts/git_update.py

```diff
@@ -0,0 +1,90 @@
+import os
+import subprocess
+import sys
+import io
+import shutil
+
+import django
+
+from ports.models import Commit
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(BASE_DIR)
+os.environ['DJANGO_SETTINGS_MODULE'] = 'MacPorts.settings'
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "MacPorts.settings")
+
+django.setup()
+
+
+def clone_repo():
+    if os.path.isdir('macports-ports'):
+        return
+    else:
+        os.system("git clone https://github.com/macports/macports-ports.git")
+
+
+def get_list_of_changed_ports(new_hash=False, old_hash=False, root=BASE_DIR):
+    os.chdir(root)
+
+    if os.path.isdir('macports-ports'):
```

Comment:
This can be resolved probably.

Reply:
Although the comments do mention `macports-ports` at multiple places in the code, I believe we can live with it since we are not trying to generalize the script anyway.